cluster/schema/schema.go

```diff
@@ -355,14 +355,6 @@ func (s *schema) MetaClasses() map[string]*metaClass {
 	return s.Classes
 }
 
-func (s *schema) clear() {
```

Comment:
**thought (non-blocking):** intended ? just double checking

Reply:
Yeah it was only used for testing, adjusted the test and now it's no longer needed